docqueries/driving_distance/dijksraDD-issue729.result

```diff
@@ -11,19 +11,19 @@ UPDATE edges AS edge_table
 SET node_count=count.sum
 FROM
         (SELECT
-        from_v,
+        start_vid,
         sum(node) AS sum
         FROM
             pgr_drivingDistance(
             'SELECT id, source, target, ST_Length(geom) AS cost FROM edges',
             ARRAY(SELECT DISTINCT source FROM edges),
             1,
             false)
-    GROUP BY from_v) AS count
-    WHERE edges.source=count.from_v
+    GROUP BY start_vid) AS count
+    WHERE edges.source=count.start_vid
     ;
 ERROR:  invalid reference to FROM-clause entry for table "edges"
-LINE 14:     WHERE edges.source=count.from_v
+LINE 14:     WHERE edges.source=count.start_vid
```

Comment:
Clearly there is something worng.
Is it an original issue from the pgRouting repo?
If it is, then there is something wrong with the original. aka someone, probably me, did what you are doing without really looking.